servers/src/proto/journal/lineage.proto

```diff
@@ -18,21 +13,11 @@ message LineageEntry {
   optional int64 id = 1;
   repeated int64 input_files = 2;
   repeated int64 output_file_ids = 3;
-  repeated LineageFileState output_file_states = 4;
```

Comment:
why remove the gap in journal.proto but leave it here? (either way is fine with me, but lets be consistent)